content/news/2021-09-13-darktable-3.6.1-released/index.md

```diff
@@ -0,0 +1,81 @@
+---
+author: Pascal Obry
+date: 2021-09-13 10:00:00+00:00
+layout: post
+title: "darktable 3.6.1 released"
+lede: vassiviere.jpg
+lede_author: <a href="http://photos.obry.net">Pascal Obry</a>
+tags:
+  - announcement
+  - darktable release
+---
+
+We're proud to announce the new feature release of darktable, 3.6.1!
+
+The github release is here: [https://github.com/darktable-org/darktable/releases/tag/release-3.6.1](https://github.com/darktable-org/darktable/releases/tag/release-3.6.1).
+
+As always, please don't use the autogenerated tarball provided by
+github, but only our tar.xz file. the checksums are:
+
+```
+$ sha256sum darktable-3.6.1.tar.xz
+a2bfc7c103b824945457a9bfed9e52f007fa1d030f9dbcb3ff0327851be42d14 darktable-3.6.1.tar.xz
+$ sha256sum darktable-3.6.1.5.dmg
+0fa0cd49971fdf76f690be095600250ad7777c2317f969c7135f035bc7b5f0ee darktable-3.6.1.5.dmg
+$ sha256sum darktable-3.6.1.exe
+34b4c1748ffe7ac66c3c79ad474b86a7aaa01ba296f8cd21929fe8024ebf3262 darktable-3.6.1-win64.exe
+```
+
+When updating from the currently stable 3.4.x series, please bear in
+mind that your edits will be preserved during this process, but the new
+library and configuration will not be usable with 3.4.x any more.
+
+You are strongly advised to take a backup first.
+
+#### Important note: to make sure that darktable can keep on supporting the raw file format for your camera, *please* read [this post](https://discuss.pixls.us/t/raw-samples-wanted/5420?u=lebedevri) on how/what raw samples you can contribute to ensure that we have the *full* raw sample set for your camera under CC0 license!
+
+## Bug Fixes
+
+- Fix module expand on quick access panel when enabling/disabling module.
+
+- Fix a possible crash in color calibration module.
+
+- Fix an inconsistency between the CPU & GPU path in color calibration module.
+
+- Fix possible issue in OpenCL Bradford non-linear code in color
+  calibration which leads to wrong color rendering.
+
+- Fix Windows paths handling and encoding.
+
+- Fix possible issue in exiv2 when trying to write empty strings.
+
+- Fix by-scale button in export module.
+
+- Fix crash when renaming a drawn mask.
+
+- Minor UI fix in duplicate module.
+
+- Fix possible wrong rendering of the color picker
+  tooltip. (workaround for a possible Gtk issue).
+
+- Fix filename variable used when importing new pictures.
+
+- Fix noise profile for RICOH GR III.
+
+- Fix some memory leaks.
+
+## RawSpeed changes
+
+## Camera support, compared to 3.6
+
```

Comment:
Should we remove these empty headers?

Reply:
oh actually, perhaps just the RawSpeed one

Reply:
Yes.